wp-content/themes/core/assets/pcss/global/reset.pcss

```diff
@@ -63,6 +63,12 @@ picture {
 	max-width: 100%;
 }
 
+/**
+ * @todo Need to ensure we can properly enqueue this reset to prevent pattern
+ * previews from disappearing in the editor. Although, they don't appear to
+ * currently be breaking, maybe something to keep an eye on.
+ * @see https://github.com/wpcomvip/careforth/pull/14
+ */
 iframe
```

Comment:
I know this isn't part of your edits, but are we missing a comma after iframe?  All the other CSS properties are listed single line with a comma after, and this one isn't...